sample/bootstrap-xds.yaml

```diff
@@ -32,3 +32,14 @@ static_resources:
         port_value: 18000
     http2_protocol_options: {}
     name: xds_cluster
+layered_runtime:
+  layers:
+    - name: runtime-0
```

Comment:
Can the RTDS API describe the layers fully without having to define the layers in the config file as well?

Reply:
Does RTDS support collections of layers? E.g. can you retrieve all layers without naming them?

Reply:
i know RTDS supports multiple layers, but I don't think the API itself supports retrieval of more than 1 layer at a time. will need to double check :)

Reply:
The API might only support retrieving single layers at a time. E.g. when I specify 2 RTDS layers in my Envoy config file:
```
layered_runtime:
  layers:
    - name: runtime-0
      rtds_layer:
        rtds_config:
          api_config_source:
            api_type: GRPC
            grpc_services:
              envoy_grpc:
                cluster_name: xds_cluster
        name: runtime-0
    - name: runtime-1
      rtds_layer:
        rtds_config:
          api_config_source:
            api_type: GRPC
            grpc_services:
              envoy_grpc:
                cluster_name: xds_cluster
        name: runtime-1
```
I see the following outputted by the RTDS server:
```
2019/09/17 09:04:12 stream 6 open for type.googleapis.com/envoy.service.discovery.v2.Runtime
2019/09/17 09:04:12 stream 7 open for type.googleapis.com/envoy.service.discovery.v2.Runtime
2019/09/17 09:04:12 respond type.googleapis.com/envoy.service.discovery.v2.Runtime[runtime-0] version "" with version "v0"
2019/09/17 09:04:12 respond type.googleapis.com/envoy.service.discovery.v2.Runtime[runtime-1] version "" with version "v0"
2019/09/17 09:04:12 open watch 7 for type.googleapis.com/envoy.service.discovery.v2.Runtime[runtime-0] from nodeID "test-id", version "v0"
2019/09/17 09:04:12 open watch 8 for type.googleapis.com/envoy.service.discovery.v2.Runtime[runtime-1] from nodeID "test-id", version "v0"
```

Reply:
Oohhh!!! It looks like ADS does support retrieving multiple layers, as running ADS with 2 RTDS layers, I see:

2019/09/17 09:29:25 respond type.googleapis.com/envoy.service.discovery.v2.Runtime[runtime-1 runtime-0] version "v1" with version "v2"
2019/09/17 09:29:25 open watch 14 for type.googleapis.com/envoy.service.discovery.v2.Runtime[runtime-1 runtime-0] from nodeID "test-id", version "v2"

Reply:
Yeah, that's how ADS is supposed to work. It should batch requests. Thanks for confirming that works in envoy.